src/groups/mqb/mqbblp/mqbblp_clusterqueuehelper.cpp

```diff
@@ -4485,6 +4486,7 @@ ClusterQueueHelper::ClusterQueueHelper(
 , d_numPendingReopenQueueRequests(0)
 , d_primaryNotLeaderAlarmRaised(false)
 , d_stopContexts(allocator)
+, d_counterOfUnconfirmed(0)
```

Comment:
Would prefer `bsl::nullptr_t()`, `nullptr` or `NULL` here, because `0` is confusing. Looking at this line, It's possible to assume that this line sets the `0 initial value` of the counter, but here it sets the `0 pointer` to a missing parent.